core/modules/widgets/navigator.js

```diff
@@ -122,12 +126,17 @@ NavigatorWidget.prototype.replaceFirstTitleInStory = function(storyList,oldTitle
 };
 
 NavigatorWidget.prototype.addToStory = function(title,fromTitle) {
-	if(this.storyTitle) {
-		this.wiki.addToStory(title,fromTitle,this.storyTitle,{
-			openLinkFromInsideRiver: this.getAttribute("openLinkFromInsideRiver","top"),
-			openLinkFromOutsideRiver: this.getAttribute("openLinkFromOutsideRiver","top")
-		});
-	}
+	this.wiki.addToStory(title,fromTitle,this.storyTitle,{
```

Comment:
This PR will invalidate PR #4200 "Remove dependency on wiki.js for story-startup and navigator" where we don't have a decision yet. 

PR #4200 tries to fix a problem, that is also discussed in issue: #4180. Which links to many other issues, which are closed, but _not_ resolved. 

Proposal: make navigator.js + friends "multi - story" aware. #4354 also discusses several elements that this PR touches. ... I would like to discuss the main mechanism first.